go/host/rpc/enclaveclient/enclave_client.go

```diff
@@ -8,36 +8,26 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/obscuronet/go-obscuro/go/common/retry"
-
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/obscuronet/go-obscuro/go/common"
 	"github.com/obscuronet/go-obscuro/go/common/log"
+	"github.com/obscuronet/go-obscuro/go/common/retry"
 	"github.com/obscuronet/go-obscuro/go/common/rpc"
 	"github.com/obscuronet/go-obscuro/go/common/rpc/generated"
 	"github.com/obscuronet/go-obscuro/go/common/tracers"
 	"github.com/obscuronet/go-obscuro/go/config"
 	"github.com/obscuronet/go-obscuro/go/responses"
-
-	"google.golang.org/grpc"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/credentials/insecure"
 
 	gethcommon "github.com/ethereum/go-ethereum/common"
 	gethlog "github.com/ethereum/go-ethereum/log"
 	gethrpc "github.com/ethereum/go-ethereum/rpc"
+	grpc "google.golang.org/grpc"
 )
 
-// Client implements enclave.Enclave and should be used by the host when communicating with the enclave via RPC.
-type Client struct {
-	protoClient generated.EnclaveProtoClient
-	connection  *grpc.ClientConn
-	config      *config.HostConfig
-	logger      gethlog.Logger
-}
-
-func NewClient(config *config.HostConfig, logger gethlog.Logger) *Client {
+func NewEnclaveRPCClient(config *config.HostConfig, logger gethlog.Logger) common.Enclave {
```

Comment:
It's hard to follow the reason for these changes.
Stuff has moved around, but is there any change?
Why were the changes/ or moving necessary?